spec/services/job_processor_spec.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+class TestJob < ActiveJob::Base
```

Comment:
We don't have an ApplicationJob yet.

Do you remember https://github.com/openfoodfoundation/openfoodnetwork/pull/9687/files/b1e4c2c022bbb5d4eaca4f017251bf2816a521c3#diff-a96d254bf8e3bf6fb5c3e7eda200acc13c867cb7ecab5d94dcd62216035b4c33?

I followed the same logic that we don't need an empty class. At least not in this PR. I do think that we should open a separate PR to introduce the ApplicationJob to conform to standards. It will make it easier to use generators and will be less work with Rubocop. That's why I see it as a todo and put it there. Maybe I just open that PR now to make this discussion obsolete. :stuck_out_tongue:

Reply:
See
- https://github.com/openfoodfoundation/openfoodnetwork/pull/10283

Reply:
> We don't have an ApplicationJob yet.

🤦 . I assumed we did because Rubocop brought it up.

I agree, thanks for creating that!